application/views/logbookadvanced/index.php

```diff
@@ -51,7 +51,8 @@
 			\"dok\":{\"show\":\"true\"},
 			\"wwff\":{\"show\":\"true\"},
 			\"sig\":{\"show\":\"true\"},
-			\"continent\":{\"show\":\"true\"}
+			\"continent\":{\"show\":\"true\",
```

Comment:
here is missing a bracket

```suggestion
			\"continent\":{\"show\":\"true\"},
```